hsapp/app.go

```diff
@@ -222,22 +223,55 @@ func (a *App) render() {
 	a.TextureLoader.ResumeLoadingTextures()
 }
 
+// please note, that this steps will not affect app language
+// it will only load an appropriate glyph ranges for
+// displayed text (e.g. for string/font table editors)
 func (a *App) setupFonts() {
-	// Note: To support other languages we'll have to do something with glyph ranges here...
-	// ranges := imgui.NewGlyphRanges()
-	// rb := imgui.NewFontGlyphRangesBuilder()
-	// rb.AddRanges(imgui.CurrentIO().Fonts().GlyphRangesJapanese())
-	// rb.AddRanges(imgui.CurrentIO().Fonts().GlyphRangesChineseSimplifiedCommon())
-	// rb.AddRanges(imgui.CurrentIO().Fonts().GlyphRangesCyrillic())
-	// rb.AddRanges(imgui.CurrentIO().Fonts().GlyphRangesKorean())
-	// rb.BuildRanges(ranges)
-	// imgui.CurrentIO().Fonts().AddFontFromFileTTFV("NotoSans-Regular.ttf", 17, 0, imgui.CurrentIO().Fonts().GlyphRangesJapanese())
-	imgui.CurrentIO().Fonts().AddFontFromMemoryTTF(hsassets.FontNotoSansRegular, 17)
-	a.fontFixed = imgui.CurrentIO().Fonts().AddFontFromMemoryTTF(hsassets.FontCascadiaCode, 15)
-	a.fontFixedSmall = imgui.CurrentIO().Fonts().AddFontFromMemoryTTF(hsassets.FontCascadiaCode, 12)
-	a.diabloRegularFont = imgui.CurrentIO().Fonts().AddFontFromMemoryTTF(hsassets.FontDiabloRegular, 15)
-	a.diabloBoldFont = imgui.CurrentIO().Fonts().AddFontFromMemoryTTF(hsassets.FontDiabloBold, 30)
-	imgui.CurrentStyle().ScaleAllSizes(1)
+	// get font manager
+	fonts := g.Context.IO().Fonts()
+
+	// create glyph ranges
+	ranges := imgui.NewGlyphRanges()
+
+	// create glyph ranges builder
+	builder := imgui.NewFontGlyphRangesBuilder()
+
+	// add default ranges
+	builder.AddRanges(fonts.GlyphRangesDefault())
```

Comment:
There are a bunch of comments in `setupFonts` that seem a bit redundant (the method names already describe what's in the comment).

Reply:
60acb24